app/src/main/java/net/gsantner/markor/ui/AttachImageOrLinkDialog.java

```diff
@@ -216,6 +236,23 @@ public void onFsViewerConfig(FilesystemViewerData.Options dopt) {
         return builder.show();
     }
 
+    private static void copyFileToZimPageFolder(File fileToBeCopied, File currentWorkingFile, Activity activity) {
```

Comment:
Please keep Zim specific functions out of the dialog class. Use a common Zim class if required, but not spread it all over place.

Reply:
Changed: extracted to a utility class.